pxtwinrt/winrt.ts

```diff
@@ -84,23 +84,52 @@ namespace pxt.winrt {
             });
     }
 
+    export function releaseAllDevicesAsync(): Promise<void> {
+        if (!isWinRT()) {
+            return Promise.resolve();
+        }
+
+        return Promise.resolve()
+            .then(() => {
+                if (packetIO) {
+                    pxt.log(`disconnecting packetIO`);
+                    return packetIO.disconnectAsync();
+                }
+                return Promise.resolve();
+            })
+            .catch((e) => {
+                pxt.reportError("winrt_device", `error disconnecting packetIO: ${e.message}`);
```

Comment:
Use lf

Reply:
reportException logs the stacktrace as well

Reply:
Not meant to be user facing, this is just for us